gdk4/Gir.toml

```diff
@@ -355,3 +354,10 @@ manual_traits = ["ToplevelExtManual"]
     [[object.function]]
     name = "show_window_menu"
     manual = true # event is a fundemental type
+
+[[object]]
```

Comment:
there's a test to ensure the objects are ordered alphabetically on the gir file

Reply:
ok, thanks